rolling/distribution.yaml

```diff
@@ -2683,6 +2703,16 @@ repositories:
       url: https://github.com/OUXT-Polaris/ouxt_common.git
       version: master
     status: developed
+  param_util:
+    doc:
+      type: git
+      url: https://github.com/hatchbed/param_util.git
```

Comment:
> So, is rosparam_<some acceptable modifier> a good approach?

Yes, that or `<prefix>_rosparam` would work.
But the modifier/prefix, should be something not generic, i.e. avoid "tool" or "kit".

Of the ones you listed, `choreo_param_helpers` and `rtt_rosparam` are good examples, as it's clear to what they refer to.
Example that could work for you: `hatchbed_rosparam` (you may find something better :smile: )

Reply:
That sounds good to me as well